rollup.config.js

```diff
@@ -8,14 +8,15 @@ import fs from 'fs'
 import path from 'path'
 import genTypes from './src/scripts/gen-svelte-types.js'
 import genBindings from './src/scripts/gen-react-bindings.js'
+import { getSvelteFiles } from './src/scripts/common.js'
 
 // Entry points are all our Svelte components + the react bindings for those
 // components.
 const COMPONENTS_FOLDER = path.resolve('./', 'src', 'components')
-const inputs = fs
-  .readdirSync(COMPONENTS_FOLDER, { withFileTypes: true })
-  .filter((f) => f.isDirectory())
-  .map((f) => path.join(COMPONENTS_FOLDER, `${f.name}/${f.name}.svelte`))
+const inputs = []
+
+for await (const file of getSvelteFiles(COMPONENTS_FOLDER, false))
+  inputs.push(file)
```

Comment:
My only nit would be that `{}` brackets around the `for` loop block would be nice. ;-)

Reply:
Booooo :laughing: (I've added them for you)

Reply:
Ha ha! Thank you. :-)